packages/flutter/lib/src/material/i18n/material_pt.arb

```diff
@@ -13,6 +15,8 @@
   "pageRowsInfoTitle": "$firstRow–$lastRow de $rowCount",
   "pageRowsInfoTitleApproximate": "$firstRow–$lastRow de cerca de $rowCount",
   "rowsPerPageTitle": "Linhas por página:",
+  "@selectedRowCountTitleZero": { "notUsed": "TODO: translate or explain why omitted" },
```

Comment:
Only an fooOther resource is required, the Zero, One, Two, Few, and Many variations are optional.

Reply:
Done.